src/fsharp/NameResolution.fs

```diff
@@ -2014,8 +2014,15 @@ let rec ResolveExprLongIdentInModuleOrNamespace (ncenv:NameResolver) nenv (typeN
                 else 
                     NoResultsOrUsefulErrors
 
-            AtMostOneResult id.idRange ( tyconSearch +++   moduleSearch +++ raze (UndefinedName(depth,FSComp.SR.undefinedNameValueConstructorNamespaceOrType,id,NoPredictions)))
-
+            match tyconSearch +++ moduleSearch with
```

Comment:
AtMostOneResult was moved to line 2025. but thanks for the test report. I will try to fix that.